libs/workspace-manager/src/lib/features/workspace-manager/workspace-manager.component.scss

```diff
@@ -0,0 +1,46 @@
+@use '@angular/material' as mat;
+@use 'material/colors' as colors;
+
+:host {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.workspace-manager-header {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  height: 150px;
+  background-color: mat.get-color-from-palette(colors.$primary);
+  color: mat.get-color-from-palette(colors.$primary, '300-contrast');
+  padding: 0 10px;
+
+  h1 {
+    display: flex;
+    align-items: center;
+    gap: 5px;
+    font-size: 18px;
+
+    img {
+      width: 50px;
+      height: 50px;
+    }
+  }
+
+  nav {
+    display: flex;
+    justify-content: space-around;
+
+    a.active {
```

Comment:
please don't change the colors of basic elements try to keep them as close to material as you can (use the primary/accent/warn classes)

Reply:
Not sure I understand, I'm using colors from the material palette

Reply:
if you want to change the color of A tag you can override the current css globally:

`.mat-mdc-button.mat-unthemed` to be 

```
a.mat-mdc-button.mat-unthemed {
   ....
   &.active {
    ....
   }
}
```

calls this file "buttons.scss" and import it in the "theme.scss"